moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryDataTrackerSignals.js

```diff
@@ -27,20 +27,18 @@ const ProjectSummaryDataTrackerSignals = ({ project }) => {
   const signals = useProjectSignals(project);
 
   return (
-    <>
```

Comment:
I removed the extra fragment wrapping around the Grid, and so the indentation changed.